mender/values.yaml

```diff
@@ -10,16 +10,23 @@ global:
     URL: mongodb://mongodb
   nats:
     URL: nats://nats:4222
+  storage: "aws"
   s3:
     AWS_URI: ""
     AWS_EXTERNAL_URI: ""
-    AWS_BUCKET: mender-artifact-storage
+    DEPLOYMENTS_STORAGE_BUCKET: mender-artifact-storage
```

Comment:
Please see my comments on the documentation.

Reply:
Done!